catalog/core/catalog-core-commands/pom.xml

```diff
@@ -134,6 +134,12 @@
             <artifactId>catalog-transformer-zip</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
```

Comment:
Did coverage go up or down? It seems hard to believe it held steady with a new class and new test class for it.